java/client/src/main/java/glide/api/commands/HashCommands.java

```diff
@@ -19,6 +19,13 @@ public interface HashCommands {
      * @param field The field in the hash stored at <code>key</code> to retrieve from the database.
      * @return The value associated with <code>field</code>, or <code>null</code> when <code>field
      *     </code> is not present in the hash or <code>key</code> does not exist.
+     * @example
+     *     <pre>
+     * String payload = client.hget("my_hash", "field1").get();
+     * assert payload.equals("value");
+     * String payload = client.hget("my_hash", "nonexistent_field").get();
```

Comment:
Should we split and add one more `@example` tag?

Reply:
I have been doing it this way for a while what is the preference? @acarbonetto